### ISSUE TITLE
Policy IPolicyType arguments can only be one RDF Term

### ISSUE THREAD

**Issue by woutslabbinck** [eyereasoner/Koreografeye#10]
https://github.com/eyereasoner/Koreografeye/blob/3fa0ed8c2a8aa4759ad40b84b66d64c419888a8d/src/policy/PolicyPlugin.ts#L14-L16

Due to the `IPolicyType` type having for the arguments only the possibility to have an RDF.Term with cardinality one, the `extractPolicies` algorithm will pick a random RDF.Term for the arguments where in the conclusion of  an N3 rule where this cardinality was N.

My proposal is to fix this by introducing a breaking change that will fix this problem:
```ts
export type IPolicyType = {
    node: N3.NamedNode | N3.BlankNode;
    path: string;
    policy: string;
    target: string;
    mainSubject: string;
    origin: string;
    order: number;
    args: {
        [key: string]: RDF.Term[];
    };
};
```

For this, `extractPolicies` must also be changed to make it possible to add a list of RDF.Term[]. 

Furthermore, there will be no undefined anymore since it can now have the empty list.

**Comment by phochste** [comment]
I need to check this first for unintended consequences. The rules can be fired multiple times and we don't want that the `args`[key] array will get filled with bogus information ..due to this multiple firing of rules.

**Comment by phochste** [comment]
fixed in release 0.4.0
